spotdl/search/provider.py

```diff
@@ -266,3 +268,45 @@ def search_and_get_best_match(songName: str, songArtists: List[str],
     # ! In theory, the first 'TUPLE' in sortedResults should have the highest match
     # ! value, we send back only the link
     return sortedResults[0][0]
+
+
+def get_song_lyrics(songName: str, songArtists: List[str]) -> str:
+    """
+    `str` `songName` : name of song
+
+    `list<str>` `songArtists` : list containing name of contributing artists
+
+    RETURNS `str`: Lyrics of the song.
+
+    Gets the metadata of the song.
+    """
+
+    headers = {
+        'Authorization': 'Bearer alXXDbPZtK1m2RrZ8I4k2Hn8Ahsd0Gh_o076HYvcdlBvmc0ULL1H8Z8xRlew5qaG',
+    }
+    apiSearchUrl = 'https://api.genius.com/search'
+    searchQuery = f'{songName} {", ".join(songArtists)}'
+
+    apiResponse = get(
+        apiSearchUrl,
+        params={'q': searchQuery},
+        headers=headers
+    ).json()
+
+    songId = apiResponse['response']['hits'][0]['result']['id']
+    songApiUrl = f'https://api.genius.com/songs/{songId}'
+
+    apiResponse = get(
+        songApiUrl,
+        headers=headers
+    ).json()
+
+    songUrl = apiResponse['response']['song']['url']
+
+    geniusPage = get(songUrl)
+    soup = BeautifulSoup(geniusPage.text, 'html.parser')
+    lyrics = soup.select_one('div.lyrics').get_text()
+
+    return lyrics.strip()
+
+    
```

Comment:
```suggestion
    return lyrics.strip()
```

![image](https://user-images.githubusercontent.com/42355410/110792273-77576c00-8273-11eb-9ba4-757a874776d5.png)